codecademy/classes_second_part.py

```diff
@@ -39,3 +38,29 @@ def __init__(self, x, y, z):
     def __repr__(self):
         return "(%d, %d, %d)" % (self.x, self.y, self.z)
 
+# nicolas13sochi part
+
+
+class Car9(object):
+    condition = "new"
```

Comment:
move this inside constructor, make this an object field `self.condition = "new"`